content/community/chat.md

```diff
@@ -9,26 +9,29 @@ aliases:
 
 The XSF hosts a variety of chat rooms and all archives are available online. Participation in the XSF's discussion channels is subject to the [Community Code of Conduct](https://xmpp.org/extensions/xep-0458.html).
 
-### Archives
+## Archives
 
-Logs of XSF public chatrooms are available at [logs.xmpp.org](https://logs.xmpp.org/).
+All logs of public XSF chatrooms are available at [logs.xmpp.org](https://logs.xmpp.org/).
```

Comment:
What's the intention behind this change?

Reply:
I think my formulation is more precise